var/spack/repos/builtin/packages/libpcap/package.py

```diff
@@ -32,3 +32,6 @@ class Libpcap(AutotoolsPackage):
     url      = "http://www.tcpdump.org/release/libpcap-1.8.1.tar.gz"
 
     version('1.8.1', '3d48f9cd171ff12b0efd9134b52f1447')
+
+    depends_on('flex', type='build')
```

Comment:
are these required for both packages, or just tcptrace ?

Reply:
I had to add it for libpcap to build. Whoever originally made the package probably had build tools in their path.

``` bash
1 error found in build log:
     [ ... ]
     108   checking whether we have DAG API headers... no
     109   checking whether we have Septel API headers... no
     110   checking whether we have Myricom Sniffer API... no
     111   checking whether TurboCap is supported... no
     112   checking for flex... no
     113   checking for lex... no
  >> 114   configure: error: Neither flex nor lex was found.
```

``` bash
1 error found in build log:
     [ ... ]
     114   checking lex library... none needed
     115   checking whether yytext is a pointer... no
     116   checking for capable lex... yes
     117   checking for bison... no
     118   checking for byacc... no
     119   checking for capable yacc/bison... insufficient
  >> 120   configure: error: yacc is insufficient to compile libpcap.
     121    libpcap requires Bison, Berkeley YACC, or another YACC compatible with them.
```